GeneralSQL/Error.py,sha256=kBrYZ17K0Jps9-DY3QFG_tLa9GOrLBJvoTLv8wwxfMk,2321
GeneralSQL/__init__.py,sha256=BCf-6GB0Fa_L6JaZRn4eGi2PBfwicxjlsw23OnDCK_Q,2155
GeneralSQL/Template/__init__.py,sha256=F80meUoOqQIOiYxE9ryCJzTMqFPBOwajkaf6Gk1e7Zk,1431
GeneralSQL/Template/base.py,sha256=oQTj7lN5JmlD3l3fG3-pm3dD8lPd0e6T1H5b0yOxToE,3024
GeneralSQL/db/Jmysql.py,sha256=V6SLrgY1AWSebAvqZsSrQuJUmQGpVvHbSaX0fIrTTn4,15681
GeneralSQL/db/__init__.py,sha256=PgWm-6m5ZDVt2vjNOtrgO7sQYzXa6A0p8Y2RnoZNTl0,57
GeneralSQL-1.1.8.dist-info/METADATA,sha256=OYMBEhGZhtMuH-P2cXfgNDqKJKgQ6PLQL45-3M2z3Y8,42392
GeneralSQL-1.1.8.dist-info/WHEEL,sha256=oiQVh_5PnQM0E3gPdiz09WCNmwiHDMaGer_elqB3coM,92
GeneralSQL-1.1.8.dist-info/top_level.txt,sha256=N28y8Tp5KNt4vFIdlg6UXzrY9QGtXwgK-oqRvdRBuh8,11
GeneralSQL-1.1.8.dist-info/RECORD,,
